config/default.json

```diff
@@ -3,8 +3,8 @@
     "client": "postgresql",
     "connection": {
       "database": "thesis",
-      "user": "postgres",
-      "password": "postgres",
+      "user": "Nicolaou",
```

Comment:
This looks like you are setting the default user as your last name? Is this on purpose?

Reply:
For my local sign in I log in as myself instead of as postgres. was planning on merging in another request and overwriting that in the master.

Reply:
Ok yea because this commit basically is saying this is a requirement for everyone. In the future please confirm that code is generic for all devs.